hosts/hedgehog/release

```diff
@@ -0,0 +1 @@
+19.03
```

Comment:
it's 19.03 by default -> https://github.com/kalbasit/shabka/blob/master/.release.

Reply:
Yes, indeed. But once the default changes, I want to be able to keep my version without having to worry about the default. Plus it allows me to just change the version to `unstable` for example without having to recreate the file.